web-registry/src/routes.tsx

```diff
@@ -87,135 +89,104 @@ type RouterParams = {
 export const getRoutes = ({ reactQueryClient }: RouterParams): RouteObject[] =>
   createRoutesFromElements(
     <Route element={<RegistryLayout />}>
-      <Route
-        path="/"
-        element={<Home />}
-        loader={homeLoader({
-          queryClient: reactQueryClient,
-        })}
-      />
-      <Route path="verify-email" element={<VerifyEmail />} />
-      <Route path="add" element={<Additionality />} />
-      <Route path="signup" element={<Signup />} />
-      <Route path="certificate" element={<CertificatePage />} />
-      <Route path="project/wilmot/admin" element={<Seller />} />
-      <Route path="buyers" element={<BuyersPage />} />
-      <Route path="create-methodology" element={<CreateMethodology />} />
-      <Route
-        // TODO: thould this route be moved to /credit-classes?
-        path="create-credit-class"
-        element={<CreateCreditClassInfo />}
-      />
-      <Route path="land-stewards" element={<LandStewards />} />
-      <Route
-        path="methodology-review-process"
-        element={<MethodologyReviewProcess />}
-      />
-      <Route
-        path="projects/:page"
-        element={<Projects />}
-        loader={projectsLoader({
-          queryClient: reactQueryClient,
-        })}
-      />
-      <Route path="project/:projectId" element={<Project />} />
-      <Route
-        path="post-purchase/:projectId/:walletId/:name"
-        element={<PostPurchase />}
-      />
-      <Route path="ecocredits" element={<KeplrRoute component={Dashboard} />}>
+      <Route path="/" element={<Outlet />} errorElement={<ErrorPage />}>
         <Route
-          path="portfolio"
-          element={<KeplrRoute component={MyEcocredits} />}
+          index
+          element={<Home />}
+          loader={homeLoader({
+            queryClient: reactQueryClient,
+          })}
```

Comment:
this has been the main change to establish a first general boundary for the errors in the router. To do this, a new Route component has been introduced, below the root Route corresponding to the layout (to preserve the layout), and wrapping all the other routes of the application.